src/Ledger/Utxow.lagda

```diff
@@ -216,8 +219,12 @@ the \List \Data argument we provide would be required.
 In Figure~\ref{fig:rules:utxow}, the line
 \inputHashes~\subseteqfield~\txdatsHashes compares two inhabitants of
 \PowerSet~\DataHash.  In the original Alonzo spec, these two terms would
-have inhabited \PowerSet~(\Maybe~\DataHash), where a \nothing is thrown out.
+have inhabited \PowerSet~(\Maybe~\DataHash), where a \nothing is thrown out
+(see~\cite[Sec.~3.1]{alonzo-ledger-spec}).
 In original spec, however, the right-hand side (\txdatsHashes) could never
 contain \nothing, hence the left-hand side (\inputHashes) could never
 contain \nothing.
-
+% TODO: Say what "original spec" refers to?
```

Comment:
I feel like we can just remove the word 'original'. It's just the Alonzo spec.

Reply:
Okay, I'll remove the word "original" in front of "Alonzo."  But in the next sentence, the word "original" is ambiguous.  It says, "In the original spec, however,..." so presumably that's a reference to Shelley...?

Reply:
In line 224, 'original spec' also refers to Alonzo. Shelley didn't have this line.

Reply:
Okay, thanks.  In that case, the "however" connective doesn't make sense.  I'll rewrite the sentence.